### DISCUSSION TITLE
Import with Gocardless: any way to save settings?

### DISCUSSION THREAD

**Discussion by mikkelnl** [firefly-iii/firefly-iii#9141] [Firefly III]
Hi all,

Just started experimenting, and I got the gocardless connection working with the data importer. One thing I was wondering if this is normal: do I need to enter the secret and api key (and the follow up screens where I allow access for gocardless to my bank) every time I use the data importer? Or should this gocardless connection be saved somewhere?

**Comment by JC5** [comment]
You can set them as environment variables, it's in the docs somewhere :)
